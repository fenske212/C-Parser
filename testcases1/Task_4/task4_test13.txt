S043eSc -> A043eSc B043eSc *
S043eSc -> C043eSc D043eSc E043eSc *
A043eSc -> B043eSc *
B043eSc -> C043eSc D043eSc *
C043eSc -> E043eSc *
D043eSc -> S043eSc *
E043eSc -> B043eSc A043eSc *
E043eSc -> a043eSc b043eSc *
#
