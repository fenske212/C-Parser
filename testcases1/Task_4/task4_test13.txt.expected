FOLLOW(S043eSc) = { $, a043eSc }
FOLLOW(A043eSc) = { $, a043eSc }
FOLLOW(B043eSc) = { $, a043eSc }
FOLLOW(C043eSc) = { a043eSc }
FOLLOW(D043eSc) = { $, a043eSc }
FOLLOW(E043eSc) = { $, a043eSc }
